deploy/manifests/controller.yaml

```diff
@@ -202,6 +202,61 @@ roleRef:
   kind: Role
   name: external-attacher-cfg
   apiGroup: rbac.authorization.k8s.io
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: external-resizer-runner
+rules:
+  - apiGroups: [""]
+    resources: ["persistentvolumes"]
+    verbs: ["get", "list", "watch", "update", "patch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims"]
+    verbs: ["get", "list", "watch"]
+  - apiGroups: [""]
+    resources: ["persistentvolumeclaims/status"]
+    verbs: ["update", "patch"]
+  - apiGroups: [""]
+    resources: ["events"]
```

Comment:
Do you really update or patch an event?

Reply:
Without the `update` and `patch` verbs for event resource, resizing volumes works as before.
I also checked the source code of `external-resizer` and could not find explicit lines that are updating events.

However, they are not critical and I am not 100% sure that they can be omitted without any effects, we want to keep them.

Reply:
@binoue I don't agree.  Event resources are inherently non-updatable because its purpose is to record past events.

Reply:
An `Event` object has fields of `count`, `firstTimestamp`, and `lastTimestamp`.
It seems to correspond to a series of events, not to a single event.
`external-resizer` should update `Event` if the same type of event occurs after the first one.

In our experiment, we removed `update` verb from `ClusterRole` *after* starting `external-resizer`.
I'm not sure whether such modification affects a running Pod.

Reply:
@binoue @morimoto-cybozu 
OK, I was wrong.  The event recorder in fact updates or patches event resources:
https://github.com/kubernetes/client-go/blob/master/tools/events/event_broadcaster.go#L80

Reply:
\# Though this conversation is marked as resolved, I post the detail of updating events.

`external-resizer` starts event recording by calling `StartRecordingToSink()` with `EventSinkImpl`.
- https://github.com/kubernetes-csi/external-resizer/blob/v0.4.0/pkg/controller/controller.go#L74

In the loop started by `StartRecordingToSink()`, `EventSinkImpl`'s `Patch()` is called if the count > 1.
- https://github.com/kubernetes-csi/external-resizer/blob/v0.4.0/vendor/k8s.io/client-go/tools/record/event.go#L186
- https://github.com/kubernetes-csi/external-resizer/blob/v0.4.0/vendor/k8s.io/client-go/tools/record/event.go#L190
- https://github.com/kubernetes-csi/external-resizer/blob/v0.4.0/vendor/k8s.io/client-go/tools/record/event.go#L212
- https://github.com/kubernetes-csi/external-resizer/blob/v0.4.0/vendor/k8s.io/client-go/tools/record/event.go#L238

And `EventSinkImpl.Patch()` calls REST API `Patch`.
- https://github.com/kubernetes-csi/external-resizer/blob/v0.4.0/vendor/k8s.io/client-go/kubernetes/typed/core/v1/event_expansion.go#L162
- https://github.com/kubernetes-csi/external-resizer/blob/v0.4.0/vendor/k8s.io/client-go/kubernetes/typed/core/v1/event_expansion.go#L89